rules/aws/exfiltration_s3_data_management_tampering.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2021/05/05"
+maturity = "production"
+updated_date = "2021/05/05"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = "Identifies when threat actors tamper with S3 data management in Amazon Web Services"
+false_positives = ["Unknown"]
```

Comment:
Most of the cloud rules tend to be less threat-centric and so have high occurrences of FP (or at least need context to determine). You can also base this verbiage on other AWS rules